nemo/collections/multimodal/data/videoneva/videoneva_dataset.py

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
rename to video_neva_dataset

Reply:
similar, too many copy paste from image neva, re-use some of the components?

Reply:
Agreed. Removed `tokenize` because it is identical to `neva_dataset.py` implementation. `preprocess_multimodal` must be reimplemented in `video_neva_dataset.py` because of video token integration. 

I have split `TarOrFolderVisualLoader` which processes images and videos together into `TarOrFolderImageLoader` and `TarOrFolderVideoLoader`, where `TarOrFolderImageLoader` is imported from `neva_dataset.py` and `TarOrFolderVideoLoader` is implemented in `video_neva_dataset.py`.